pulp_container/app/global_access_conditions.py

```diff
@@ -118,3 +120,17 @@ def has_distribution_perms(request, view, action, permission):
     return any(
         (request.user.has_perm(permission, distribution.cast()) for distribution in distributions)
     )
+
+
+def has_file_repo_perms(request, view, action, permission):
+    serializer = serializers.OCIBuildImageSerializer(
+        data=request.data, context={"request": request}
+    )
```

Comment:
If we cannot make this more generic, we should rename it to something like "has_image_build_file_repo_perms".

Reply:
My bad!